apps/arweave/src/ar_nonce_limiter_client.erl

```diff
@@ -21,6 +21,11 @@
 start_link() ->
 	gen_server:start_link({local, ?MODULE}, ?MODULE, [], []).
 
+%% @doc Pick a server from the list of configured VDF servers and request complete
+%% VDF sessions from them. Failures to resolve a domain of make a request are not retried.
```

Comment:
Not retrying and only asking one VDF server is okay because we'll end up retrying with a new VDF server the next time we try to apply the block and can't validate it, right?